test/e2e/README.md

```diff
@@ -0,0 +1,67 @@
+# End-to-end Tests
+
+## Dependencies
+
+Install the `e2e` package, from the project directory:
+
+```shell
+pip install -e test/e2e
+```
+
+## Running the tests
+
+### AWS
+
+From an existing cluster:
+
+```shell
+pytest test/e2e/tests -k aws --aws-env <cortex_aws_env>
```

Comment:
`-k` it's a pattern matcher pytest flag. Technically you can run all tests without any of the gcp flags and it will skip GCP tests (but they will appear as skipped). With the `-k` flag you are able to execute just a portion of the tests.

e.g. if I wanted to run the BatchAPI tests only, I could pass `-k batch`

Reply:
👍 that makes sense, thanks!